src/go/rdctl/go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/rancher-sandbox/rancher-desktop/src/go/rdctl
+
+go 1.17
+
+require (
+	github.com/inconshreveable/mousetrap v1.0.0 // indirect
+	github.com/spf13/cobra v1.3.0 // indirect
```

Comment:
Odd, why is `cobra` indirect? Could you try running `go mod tidy` again and see if it makes a difference?

Reply:
Yes, that worked. I'll add `go mod tidy` to my arsenal.